src/js/actions/ui.js

```diff
@@ -349,27 +349,51 @@ define(function (require, exports) {
         _displayConfigurationChangedHandler;
 
     /**
+     * @const
+     * @type {number}
+     */
+    var EVENT_DEBOUNCE_DELAY = 200;
+
+    /**
      * Register event listeners for UI change events, and initialize the UI.
      *
      * @return {Promise}
      */
     var beforeStartup = function () {
-        var DEBOUNCE_DELAY = 200;
-
-        var setTransformDebounced = synchronization.debounce(function (event) {
-            if (event.transform) {
-                return this.flux.actions.ui.setTransform(event.transform);
-            }
-        }, this, DEBOUNCE_DELAY, false);
+        var scrolling = false,
+            updateTransformDebounced = synchronization.debounce(function () {
+                return this.flux.actions.ui.updateTransform()
```

Comment:
Seems OK to me, but leaving a note here for @volfied in case he wants to weigh in after the fact.